src/actions/pipeline-actions.js

```diff
@@ -0,0 +1,27 @@
+import Actions from '../reducers/actions';
+
+export const setRequestInformation = data => ({
+  type: Actions.requests.setRequestInformation,
+  data,
+});
+export const requestInProgress = data => ({
+  type: Actions.requests.requestInProgress,
+  data,
+});
+
+export const setTypeOfRequest = data => ({
+  type: Actions.requests.setTypeOfRequest,
+  data,
+});
+
+export const setRequestPrice = (event) => {
+  const stringValue = event.target.value;
+  let value = parseFloat(stringValue);
```

Comment:
We could refactor this to not use let

Reply:
We actually can actually get rid of that function but thought it'd do it in another branch when we get to request pricr

Reply:
ok, just keep in mind to not use let and extra branches of logic.

Reply:
We don't need this let. We can just use a ternary (the `?` syntax)